src/styl/style.styl

```diff
@@ -3,39 +3,43 @@
  */
 
 .container {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-evenly;
-    align-items: flex-start
+  display: grid;
+  justify-content: space-evenly
+  align-items: space-evenly
+  grid-column-gap: 8px;
+  grid-row-gap: 8px;
+  grid-template-columns: repeat(auto-fit, 270px);
+  justify-items: center;
+  margin: 32px;
 }
```

Comment:
so i think flex works better for what we want, but you're the css god, so i'll trust you on this